### PR TITLE
Support to get log of initContainers in a pod

### PR DESCRIPTION
**Pull request by TxGVNN**
HI @abrochard, 

I would like to add a feature to get the log of initContainers in a pod. Hope it is useful.

Thanks!

### CODE REVIEW

File: kubel.el [hunk 1]
```diff
@@ -794,6 +806,7 @@ RESET is to be called if the search is nil after the first attempt."
    ("-n" "Tail" "--tail=")]
   ["Actions"
    ("l" "Tail pod logs" kubel-get-pod-logs)
+   ("i" "Tail pod logs (initContainer)" kubel-get-pod-logs--initContainer)
```
Comment (abrochard) [inline kubel.el:809]: the phrasing seems a bit awkward. Maybe `Tail initContainer logs`?

**Review by abrochard — CHANGES_REQUESTED** [review]
This looks great. Minor comment on phrasing. Thank you for submitting. I will merge after fix.

File: kubel.el (hunk 1 above)
Comment (TxGVNN) [reply kubel.el]: Thanks! I updated

**Review by abrochard — APPROVED** [review]

**Comment by abrochard** [comment]
Awesome!

**Pull request merged**